test/lint/pass_lint_json.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -o errexit
+set -o nounset
+
+TMP="$(mktemp -d)"
+clean() { rm -rf "$TMP"; }
+trap clean EXIT
+
+cat << 'EOF' > "$TMP/schema.json"
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "const": "foo"
```

Comment:
This schema is invalid. `const` is not present in Draft 4. That's a linter rule we should have in the future I guess. For now, can you make sure this schema is at least valid? i.e.

```suggestion
  "enum": [ "foo" ]
```

Reply:
Just added those in my rules :D